zerver/tests/test_external.py

```diff
@@ -120,8 +118,10 @@ def test_hit_ratelimits(self):
         # We actually wait a second here, rather than force-clearing our history,
         # to make sure the rate-limiting code automatically forgives a user
         # after some time has passed.
-        time.sleep(1)
+        with mock.patch('time.time', return_value=(time.time() + 1)):
+            with mock.patch('time.sleep'):
+                    time.sleep(1)
```

Comment:
Wow I guess I wasn't thinking. Since the return value for `time.time` was already mocked the test wouldn't need those two redundant lines then. Thanks for the comment, I've updated the pr. (Just in case something odd might happen, I also ran the full test locally to be sure.)